src/query.ts

```diff
@@ -245,6 +251,11 @@ export function stringifyEncodingQueryFieldDef(encQ: EncodingQuery): string {
     if (encQ.bin['maxbins']) {
       params.push({key: 'maxbins', value: encQ.bin['maxbins']});
     }
+  } else if (encQ.scale && !isEnumSpec(encQ.scale)) {
```

Comment:
This is incorrect.  Let's chat about this to design this in person. 

This will also require further testing.